NetKAN/TweakScaleCompanion-ReStockPlus.netkan

```diff
@@ -0,0 +1,31 @@
+  
+{
+    "spec_version"     : "v1.2",
+    "$kref"            : "#/ckan/spacedock/2424",
+    "$vref"            : "#/ckan/ksp-avc",
+    "identifier"       : "TweakScaleCompanion-ReStockPlus",
+    "name"             : "TweakScale Companion for the ReStockPlus AddOn",
+    "abstract"         : "Adds TweakScale /L patches for ReStockPlus.",
+    "license"          : [ "GPL-2.0", "restricted" ],
+    "release_status"   : "stable",
+    "x_netkan_force_v" : true,
+    "author": [
+        "OnlyLightMatters",
+        "LisiasT"
+    ],
+    "install" : [
+        {
+            "file" : "GameData/TweakScaleCompanion/ReStockPlus",
+            "install_to" : "GameData/TweakScaleCompanion"
+        }
+    ],
+    "resources": {
+        "homepage":   "https://forum.kerbalspaceprogram.com/index.php?/topic/192216-*",
+        "repository": "https://github.com/net-lisias-ksp/TweakScaleCompanion_ReStockPlus"
+    },
+    "depends" : [
+        { "name" : "ModuleManager", "min_version" : "4.0.3" },
+        { "name" : "TweakScale", "min_version" : "2.4.4" },
+        { "name" : "RestockPlus", "min_version" : "1.0.3" }
```

Comment:
> `TweakScaleCompanion-ReStockPlus v1.1.0.0 requires RestockPlus 1.0.3 or later but it is not listed in the index, or not available for your version of KSP.`

Identifiers are case sensitive, `Restock` should be `ReStock`. I can fix this for you if you check this box in the lower right:

![image](https://user-images.githubusercontent.com/1559108/81486894-ba0d7980-9247-11ea-955c-886535e57c4c.png)

Reply:
I found no checkbox on this page. I will fix the problem on the pull-request.